src/commands/database-size.ts

```diff
@@ -0,0 +1,28 @@
+import { default as DatabaseSize } from "../database/size";
```

Comment:
Don't rename `default`. This is the same as `import * as DatabaseSize from "../database/size";`